packages/compass-connect/src/stores/index.js

```diff
@@ -996,6 +1054,8 @@ const Store = Reflux.createStore({
     // bar, which is hidden after the instance information is loaded
     // in another plugin.
     this.StatusActions.showIndeterminateProgressBar();
+
+    await this._trackConnectionInfo();
```

Comment:
should we wait for this?

Reply:
Yeah, that’s a good point. I would probably make this 

```suggestion
    void this._trackConnectionInfo();
```

and leave this function here synchronous – there’s not much point in letting the user wait because we are doing something telemetry-related that they never interface with.